src/js/components/banner.js

```diff
@@ -0,0 +1,16 @@
+var select = require('../utils/select');
+var addClass = require('../utils/add-class');
+var removeClass = require('../utils/remove-class');
+var dispatch = require('../utils/dispatch');
+
+var headers = select('.usa-banner-header');
+headers.forEach(function (header) {
+  var click = function (e) {
```

Comment:
OK, given Shawn's comment, I guess we do want this behaviour. @maya, can you add the line above to the top of the click function?